Azure-Samples/Cognitive-Services-Voice-Assistant
# Update view when all panes are closed and Chat History Download

## Purpose
* Download button has been renamed to Save
* Clicking Save, saves the chat history in a txt file and opens ApplicationData localfolder with newly created file selected
* Updated minimum view when all panes are collapsed

## Does this introduce a breaking change?
[ ] Yes
[x] No

## Pull Request Type
What kind of change does this Pull Request introduce?
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:

## How to Test
* Initiate a conversation with bot
* Click Save, ApplicationData LocalFolder should open with the chat history txt file selected.
* Chat history txt file contains the text, timestamp, a bool value indicating if message is sent by bot.
* Close all panes and minimum view should resemble Figma design
